rs/replicated_state/src/canister_state/system_state.rs

```diff
@@ -1718,6 +1774,20 @@ pub(crate) fn push_input(
     res
 }
 
+fn call_context_manager_mut(status: &mut CanisterStatus) -> Option<&mut CallContextManager> {
```

Comment:
Do we need any warning related to not making this public, or should we maybe even inline it given that it is only used once?

Reply:
This same function comes in the "encapsulate `CallContextManager`" PR. Likely with a warning attached. But that is unnecessary, technically speaking, since the `CallContextManager::status` will become private anyway.